edubots_surveys.Rnw

```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
 
 From a qualitative point of view, these results show consensus in a few key aspects:
 \begin{itemize}
- \item Teachers want to have support from their corresponding institution. This
-   is probably due to the strict European data regulations \textcolor{black}{on data protection},
-   but also to avoid the overhead required to \textcolor{black}{sign up new students every year.
-   In general support from specific formation and access to IT help desk for specific features
-   enhances the possibility of adoption by the teachers or the possibility of a successful
-   adoption that improves the learning outcomes. This is supported by the result of the two
-   surveys: they prefer whatever "messaging" application is provided by the university, and
-   also they value sustainability as well as "officiality" of the platform, as shown in
-   Table \ref{tab:interaction}}.
+ \item Teachers want to have support from their institutions. This
+   is probably due to the strict European data regulations on data protection,
+   but also to avoid the overhead required to sign up new students every year.
+   In general, additional support with specific formation and access to IT help desk for specific features
+   enhance the possibility of a successful adoption that improves the learning outcomes. This is supported by the result of the two
+   surveys: they prefer whatever "messaging" application is provided by their university, and
```

Comment:
whatever "messaging" application is provided by  -> whatever "messaging" application provided by

Reply:
```suggestion
   surveys: they prefer whatever ``messaging'' application provided by
```